tests/parametric/test_otel_env_vars.py

```diff
@@ -177,12 +178,15 @@ def test_otel_log_level_to_debug_mapping(self, test_agent, test_library):
             resp = t.get_tracer_config()
         assert resp["dd_trace_debug"] == "true"
         # If dd_log_level is set it must be consistent with dd_trace_debug
-        assert (resp["dd_log_level"] == "debug") or (resp["dd_log_level"] is None)
+        assert (resp["dd_log_level"] == "debug") or (resp["dd_log_level"] is None) or (resp["dd_log_level"] == "null")
```

Comment:
Just like in the context.library check above, I guess that we could refactor this in the following way?

```suggestion
        assert resp["dd_log_level"] in ("debug", None, "null")
```